packages/terra-clinical-data-grid/src/DataGrid.jsx

```diff
@@ -1100,7 +1115,12 @@ class DataGrid extends React.Component {
             >
               <div
                 className={cx(['horizontal-overflow-container', { 'padded-container': fill }])}
-                ref={this.setHorizontalOverflowContainerRef}
+                ref={(ref) => {
+                  this.setHorizontalOverflowContainerRef(ref);
```

Comment:
This logic should be added to [`this.setHorizontalOverflowContainerRef `](https://github.com/cerner/terra-clinical/blob/b069773ed557b1c21ab850266334bffe3dfcc245/packages/terra-clinical-data-grid/src/DataGrid.jsx#L549)
```diff
  setHorizontalOverflowContainerRef(ref) {
    this.setHorizontalOverflowContainerRef = ref;
+    if (this.props.horizontalOverflowContainerRefCallback) {
+       horizontalOverflowContainerRefCallback(ref)
+    }
  }

```

Reply:
Updated here https://github.com/cerner/terra-clinical/pull/585/commits/9de4536895fbfa7f35601a8b54b810108df7b7da.

Reply:
Can we revert to 
```js
 ref={this.setHorizontalOverflowContainerRef}
```
with this change?

Reply:
updated here https://github.com/cerner/terra-clinical/pull/585/commits/aac516731bb55f93b6f08d36db934d9cb68a56dd.